model/src/test/kotlin/config/PackageConfigurationTest.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.Provenance
+import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+
+private fun vcsPackageConfig(id: String, revision: String, url: String) =
+    PackageConfiguration(
+        id = Identifier(id),
+        vcs = VcsMatcher(
+            type = VcsType.GIT,
+            url = url,
+            revision = revision
+        )
+    )
+
+private fun sourceArtifactConfig(id: String, url: String) =
+    PackageConfiguration(
+        id = Identifier(id),
+        sourceArtifactUrl = url
+    )
+
+class PackageConfigurationTest : WordSpec() {
+    init {
```

Comment:
Please pass `init` as a lambda to the `WordSpec` constructor to reduce nesting.

Reply:
Done.